application/src/main/java/org/togetherjava/tjbot/commands/basic/SuggestionsUpDownVoter.java

```diff
@@ -46,6 +46,10 @@ public void onMessageReceived(@NotNull MessageReceivedEvent event) {
 
         Guild guild = event.getGuild();
         Message message = event.getMessage();
+        
+        if (!message.getChannel().getType().isThread()) {
+            message.createThreadChannel("Discussion for " + message.getAuthor().getName() + "'s suggestion").queue();
+        }
```

Comment:
Have you tested this locally in your test server already? This appears to be wrong. If I read it right, it says "if someone created a thread, lets create another thread".

Reply:
Nvm. I read it wrong. I would still be interested in how it looks in practice. Can you post a screenshot?

Reply:
when the bot creates a thread it does not sends a thread creation message
https://drive.google.com/file/d/1-JGBDCSbo2IYxgn-PKY8szrBlFTbMrcZ/view?usp=sharing
the testing bot have admin perms

Reply:
Thats good to know, but could you please send a screenshot of this piece of code, your current implementation, running live? It looks like you just created another mini project to showcase it, instead of actually trying out the real code with our real project.

Reply:
Also, the code is failing the spotless check right now.

Reply:
![image](https://user-images.githubusercontent.com/73871477/179747538-04caa34b-24f9-46a0-b6e3-d044988aa0f5.png)
This is from config file:
```json
"suggestions": {
    "channelPattern": "suggestions",
    "upVoteEmoteName": "thumbsup",
    "downVoteEmoteName": "thumbsdown"
  }
```